traitsui/qt4/file_editor.py

```diff
@@ -227,6 +227,9 @@ def init(self, parent):
     def dispose(self):
         """ Disposes of the contents of an editor.
         """
+        self._model.beginResetModel()
```

Comment:
this is tangential but i don't know why `model` is private here but public in other places